framework/docs/source/contributor-tutorial-get-started-as-a-contributor.rst

```diff
@@ -48,38 +48,36 @@ Create Flower Dev Environment
 
 1. Clone the `Flower repository <https://github.com/adap/flower>`_ from GitHub:
 
-::
-
-    $ git clone git@github.com:adap/flower.git
-    $ cd flower
+       ::
 
-2. Let's create the Python environment for all-things Flower. If you wish to use
-   ``pyenv``, we provide two convenience scripts that you can use. If you prefer using
-   something else than ``pyenv``, create a new environment, activate and skip to the
-   last point where all packages are installed.
+           $ git clone git@github.com:adap/flower.git
+           $ cd flower
 
-- If you don't have ``pyenv`` installed, the following script that will install it, set
-  it up, and create the virtual environment (with :substitution-code:`Python
-  |python_full_version|` by default):
+2. Create and activate a Python virtual environment for development. One way to do this
+   is by using `pyenv <https://github.com/pyenv/pyenv>`_ and `pyenv-virtualenv
+   <https://github.com/pyenv/pyenv-virtualenv>`_. You can also optionally install a
+   specific Python version using pyenv if you haven't already installed your desired
+   version:
 
-  ::
+       ::
 
-      $ ./dev/setup-defaults.sh <version> # once completed, run the bootstrap script
+           $ pyenv install <your-python-version>
+           $ pyenv virtualenv <your-python-version> <your-env-name>
+           $ pyenv activate <your-env-name>
```

Comment:
I wonder if with the lines above we can remove this page: https://flower.ai/docs/framework/contributor-how-to-set-up-a-virtual-env.html